### ISSUE TITLE
Need New Plugin to scan accessibility issues in source code

### ISSUE THREAD

**Issue by mpnkhan** [paypal/SeLion#385]
This is a New idea to create a In-Built plugin to scan source code for Accessibility violations. 
Use axe-core open source engine to scan. This is similar to this project https://github.com/dequelabs/axe-selenium-java .

**Comment by mach6** [comment]
@mpnkhan this is outside of the current scope of SeLion. I'm curious though, what prevents someone from adding a dependency (such as one to axe-selenium-java) to their project to accomplish the same? What value would SeLion add here?

**Comment by mpnkhan** [comment]
Quick question:
Are there any plugins available for Selion? example to test security, performance etc?

**Comment by mach6** [comment]
Not that I'm aware of

**Comment by mpnkhan** [comment]
Thanks Doug for the info. 
We have nemo framework for nodejs and nemo accessibility plugin. There are similar accessibility frameworks for some of the famous testing frameworks. So i thought it would be a good idea to have one for SeLion.

Closing this for now.
